packages/core/src/Stores/notification-store.js

```diff
@@ -1569,6 +1569,7 @@ export default class NotificationStore extends BaseStore {
     };
 
     async getP2pCompletedOrders() {
+        await WS.wait('authorize');
```

Comment:
@likhith-deriv lgtm but I remember reviewing the same fix before. Is this related to the same issue?

Reply:
U can find the same code in @aizad-deriv's PR. It is a related task